xnat_tools/bids_utils.py

```diff
@@ -206,11 +220,7 @@ def assign_bids_name(connection, host, subject, session, scanIDList, seriesDescL
 
     run_number_cache = defaultdict(int)
 
-    # Detect duplicate sequences, assume they are runs
-    # seriesDescList = detect_multiple_runs(seriesDescList)
-
-    # Cheat and reverse scanid and seriesdesc lists so numbering is in the right order
-    for scanid, seriesdesc in zip(reversed(scanIDList), reversed(seriesDescList)):
+    for scanid, seriesdesc in zip(scanIDList, seriesDescList):
```

Comment:
I thought the reversed was needed in order to order things correctly, is that not true anymore?

Reply:
I doesn't make sense why the reverse was there. That was again part of the original code that I copied, but I revised the logic and it all seems to make sense with or without it. I just means it traverses in increasing or decreasing order